src/main/webapp/scripts/map.js

```diff
@@ -75,55 +75,37 @@ function initMap() {
 }
 
 function addMarker(map) {
-  /* Hard-Coded Marker Data */
-  var placeName;
-  var placeLatLng;
-  var placeCategory;
-  var placeRecommendation;
-
   /* Iterate through stored recommendations to get details */
   for (i = 0; i < recommendations.length; i++) {
-    var rec = recommendations[i]
-    placeCategory = rec[0];
-    placeName = rec[1];
-    placeLatLng = new google.maps.LatLng(rec[2], rec[3]);
-    placeRecommendation = rec[4];
+    const recommendation = recommendations[i];
```

Comment:
@raniarusly can we have a variable name, such as aRecommendation rather than recommendation, please? I was about to push the changes, as I corrected my code based on the CL, but thanks for doing it.

Reply:
I think this is pretty common, having thing : things (I personally do this a lot and find it neat)